Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/BocSelectorColumnRendererTest.cs

```diff
@@ -58,7 +59,7 @@ public void RenderTitleCellForMultiSelect ()
       var document = Html.GetResultDocument();
 
       var th = Html.GetAssertedChildElement (document, "th", 0);
-      Html.AssertAttribute (th, "class", _bocListCssClassDefinition.TitleCell + " " + _bocListCssClassDefinition.TitleCellSelector);
+      Html.AssertAttribute (th, "class", _bocListCssClassDefinition.TitleCell + " " + CssClassDefinition.RemotionThemed + " " + _bocListCssClassDefinition.TitleCellSelector);
```

Comment:
Hmmm... I don't think we should mix-and-mingle in the tests. at least not in the same test file. Please choose one and give a short reason.

Reply:
I change the usages to `_bocListCssClassDefinition.Themed` since the property is on there as well.